.github/workflows/ci.yaml

```
@@ -88,3 +231,60 @@ jobs:
+    name: clean-sticky-cache
+    runs-on: blacksmith-2vcpu-ubuntu-2204-arm
+    needs: [build, test-node, smoke, publish, cf-deploy]
+    if: always()
+    steps:
+      - name: Sticky-disk-dist
+        uses: useblacksmith/stickydisk@v1
+        with:
+          key: ${{ github.repository }}-${{ github.ref }}-dist
+          path: ./dist
+      - name: wipe-sticky-cache
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:1:20: Quote this to prevent word splitting

(shellcheck)